bench/EnumBench.php

```diff
@@ -142,14 +147,6 @@ public function benchByValue()
         }
     }
 
-    public function benchByValueAndInitialize()
-    {
-        foreach ($this->values as $value) {
-            $this->destructEnumerations();
-            Enum66::byValue($value);
```

Comment:
This (and the other `*AndInitialize`) benchmarks doesn't make any sense and they took much tooo long